projects/aca-content/src/lib/components/search/search-save/sidenav/save-search-sidenav.component.ts

```diff
@@ -42,10 +43,12 @@ export class SaveSearchSidenavComponent implements OnInit {
   savedSearchesService = inject(SavedSearchesService);
   appService = inject(AppService);
   translationService = inject(TranslationService);
+  userPreferenceService = inject(UserPreferencesService);
   item: NavBarLinkRef;
 
   private readonly manageSearchesId = 'manage-saved-searches';
   private readonly destroyRef = inject(DestroyRef);
+  private savedSearchCount = 0;
```

Comment:
could you please add empty line between private and private readonly fields?

Reply:
Added empty line